R/modify-model-field.R

```diff
@@ -178,6 +185,72 @@ remove_tags <- function(.mod, .tags) {
     .append = FALSE,
     .remove = TRUE,
     .char_value = TRUE,
+    .bool_value = FALSE
+  )
+}
+
+
+
+#' @name add_star
```

Comment:
There are a few ways to share docs for multiple function, but for consistency, take a look at how the other wrapper functions (e.g., `add_notes` and `remote_notes`) in the file do it.  If you do it that way, you want to give this docstring a more generic name and put it above `NULL` and then add docstrings to `add_star` and `remove_star` that use `@describeIn`.